setup.py

```diff
@@ -65,6 +65,7 @@
         "myst-parser[linkify]<5",
         "sphinx>=4.6,<7",
         "sphinx-basic-ng==1.0.0b2",
+        "sphinx-build-compatibility @ git+https://github.com/readthedocs/sphinx-build-compatibility",
```

Comment:
We need to wrap that into a real package and publish to PyPI. I think we can not otherwise publish `crate-docs-theme`. Uploading will fail with `Can't have direct dependency` error.

```
make upload
```
```
WARNING  Error during upload. Retry with the --verbose option for more details.
ERROR    HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/
         Can't have direct dependency: sphinx-build-compatibility@ git+https://github.com/readthedocs/sphinx-build-compatibility. See
         https://packaging.python.org/specifications/core-metadata for more information.
make: *** [Makefile:105: upload] Error 1
```

Reply:
Fixed per vendorizing with 9d1a6a8.